cmds/webboot/webboot_test.go

```diff
@@ -262,19 +269,18 @@ func TestDownloadOption(t *testing.T) {
 
 func TestCancelDownload(t *testing.T) {
 	uiEvents := make(chan ui.Event)
+	menus := make(chan string)
+
 	// InfiniteOS will take forever to download and must be cancelled.
-	keyPresses := []string{
-		// Distros selection menu
-		strconv.Itoa(distroIndex("InfiniteOS")), "<Enter>",
-		// Mirrors selection menu
```

Comment:
I'm thinking comments like these should still be included so some reading the unit test has a better idea on which menu the key presses are for.